Tasks/AzureCLIV2/azureclitask.ts

```diff
@@ -41,7 +41,7 @@ export class azureclitask {
             });
 
             var addSpnToEnvironment: boolean = tl.getBoolInput("addSpnToEnvironment", false);
-            if (!!addSpnToEnvironment && tl.getEndpointAuthorizationScheme(connectedService, true) == "ServicePrincipal") {
+            if (!!addSpnToEnvironment && tl.getEndpointAuthorizationScheme(connectedService, true).toLowerCase() == "serviceprincipal") {
```

Comment:
@lawrencegripper please bump up the task patch version

Reply:
Sure, bumped to 2.0.13